### ISSUE TITLE
Consolidate all strategy..?

### ISSUE THREAD

**Issue by Overtorment** [bitcoinjs/coinselect#43]
I'm looking for a strategy to consolidate all inputs in a single output.
Yes this completely destroys privacy but it allows to consolidate during the period of low fees to prepare for a period of high fees.
Looks like _coinselect/split_ is what might work, but asking just for any case.
Or can implement and send a PR if you approve this idea.

**Comment by Overtorment** [comment]
Up!

**Comment by junderw** [comment]
Yeah it looks like you could just create a new strategy by importing split and making sure outputs was only one and value was undefined.

PRs are welcome. It seems like something someone could easily figure out, but having it explicitly as a file might clue people in to the methodology.
